### PR TITLE
qa: add debugging failed osd-release setting

### PR DESCRIPTION
**Pull request by batrick**
See-also: https://tracker.ceph.com/issues/40773
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: qa/tasks/ceph.py [hunk 1]
```diff
@@ -1632,9 +1632,13 @@ def restart(ctx, config):
         for dmon in daemons:
             if '.' in dmon:
                 dm_parts = dmon.split('.')
-                if dm_parts[1].isdigit():
-                    if dm_parts[0] == 'osd':
+                if dm_parts[0] == 'osd':
+                    if dm_parts[1].isdigit():
                         manager.mark_down_osd(int(dm_parts[1]))
+                    elif dm_pars[1] == '*':
```
Comment (liewegas) [inline qa/tasks/ceph.py]: typo

File: qa/tasks/ceph.py [hunk 2]
```diff
@@ -1632,9 +1632,13 @@ def restart(ctx, config):
         for dmon in daemons:
```
Comment (liewegas) [inline qa/tasks/ceph.py]: i would log the daemons list.. not sure why the existing code wouldn't work.

and/or, try just listing the osds explicitly and see if that works around the problem.

**Comment by batrick** [comment]
@liewegas I found the bug. I'll run this through testing tomorrow to confirm the fix works.

**Comment by batrick** [comment]
http://pulpito.ceph.com/pdonnell-2019-08-22_15:48:46-fs:upgrade-wip-pdonnell-testing-20190822.041539-distro-basic-smithi/

**Comment by batrick** [comment]
^ upgrade tests passed. @liewegas, waiting on your approval before merge.

**Pull request merged**